backend/tests/apps/core/api/status_test.py

```
@@ -0,0 +1,32 @@
+
+        assert response.status_code == codes.ok
+        response_data = json.loads(response.content)
+
+        assert response_data == {"version": settings.RELEASE_VERSION}
+
+    @pytest.mark.parametrize("invalid_method", ["post", "put", "delete", "patch"])
+    def test_status_endpoint_invalid_method(self, invalid_method):
+        """Test that methods other than GET are not allowed."""
+        response = getattr(self.client, invalid_method)(self.url)
+
+        assert response.status_code == codes.not_allowed
```

Pylint (3.3.7)

[error] Instance of 'LookupDict' has no 'not_allowed' member

(E1101)